code/datums/records/record.dm

```diff
@@ -70,6 +70,10 @@
 	var/minor_disabilities
 	/// Fancy description of minor disabilities
 	var/minor_disabilities_desc
+	/// Physical status of this person in medical records.
+	var/p_stat
+	/// Mental status of this person in medical records.
+	var/m_stat
```

Comment:
write out the full variable, there's no reason to abbreviate like this to single letters since it'll just hurt readability in other contexts. even `phys_status` is fine, but i'd rather you just go and do `physical` and `mental` statuses